tools/esdlcomp/esdlgram.y

```diff
@@ -193,12 +193,19 @@ EspServiceStart
         CurService->tags = getClearCurMetaTags();
 
         StringBuffer minPingVer;
+        StringBuffer pingAuthFeature("none");
         for (MetaTagInfo* t = CurService->tags; t!=NULL; t = t->next)
         {
             if (streq("ping_min_ver",t->getName()))
             {
-                minPingVer.set(t->getString());
-                break;
+                if (minPingVer.isEmpty())
```

Comment:
Is this an optimization to avoid setting minPingVer if there are multiple MetaTagInfo attributes that set ping_min_ver? If so then now the first one encountered will set the value and others are ignored. Could this be changing expected behavior? Maybe it is better to throw an error or warning if we define the same attribute multiple times.

Reply:
The break statement removed with this change ensured that only the first occurrence of ping_min_ver would be used. The check for empty mostly keeps this behavior (an empty metadata value could be replaced by the first non-empty value).

For the new attribute, I considered doing the same, but decided to accept the last instance for no other reason than it seemed to be more consistent - the last definition takes precedence over anything preceding it. I don't have strong feelings one way or another.